resources/metadata/abapEnvironmentCreateTag.yaml

```diff
@@ -0,0 +1,160 @@
+    * The Cloud Foundry parameters (API endpoint, organization, space), credentials, the service instance for the ABAP service and the service key for the Communication Scenario SAP_COM_0510.
+    * Only provide one of those options with the respective credentials. If all values are provided, the direct communication (via host) has priority.
+
+spec:
+  inputs:
+    secrets:
+      - name: abapCredentialsId
+        description: Jenkins credentials ID containing user and password to authenticate to the BTP ABAP Environment system or the Cloud Foundry API
+        type: jenkins
+        aliases:
+          - name: cfCredentialsId
+          - name: credentialsId
+    params:
+      - name: username
+        type: string
+        description: User for either the Cloud Foundry API or the Communication Arrangement for SAP_COM_0510
+        scope:
+          - PARAMETERS
+          - STAGES
+          - STEPS
+        mandatory: true
+        secret: true
+        resourceRef:
+          - name: abapCredentialsId
+            type: secret
+            param: username
+      - name: password
+        type: string
+        description: Password for either the Cloud Foundry API or the Communication Arrangement for SAP_COM_0510
+        scope:
+          - PARAMETERS
+          - STAGES
+          - STEPS
+        mandatory: true
+        secret: true
+        resourceRef:
+          - name: abapCredentialsId
+            type: secret
+            param: password
+      - name: repositories
+        type: string
+        description: Specifies a YAML file containing the repositories configuration
+        scope:
+          - GENERAL
+          - PARAMETERS
+          - STAGES
+          - STEPS
+      - name: repositoryName
+        type: string
+        description: Specifies a repository (Software Components) on the SAP BTP ABAP Environment system
+        scope:
+          - PARAMETERS
+          - STAGES
+          - STEPS
+      - name: commitID
+        type: string
+        description: Specifies a commitID, for which a tag will be created
+        scope:
+          - PARAMETERS
+          - STAGES
+          - STEPS
+      - name: tagName
+        type: string
+        description: Specifies a tagName that will be created for the repositories on the SAP BTP ABAP Environment system
+        scope:
+          - PARAMETERS
+          - STAGES
+          - STEPS
+      - name: tagDescription
+        type: string
+        description: Specifies a description for the created tag
+        scope:
+          - PARAMETERS
+          - STAGES
+          - STEPS
+      - name: createTagForAddonProductVersion
+        type: bool
+        description: Specifies a tagName that will be created for the repositories on the SAP BTP ABAP Environment system
```

Comment:
it does not specify a name, it triggers something -> bool vs. string

Reply:
fixed